app/src/main/java/com/example/poste/models/Post.java

```diff
@@ -79,6 +97,7 @@ public String getUrl() {
      */
     public String getId() { return id; }
 
+    public ArrayList<String> getTags() {return tags; }
```

Comment:
Considering the NPE risk mentioned above, I'd suggest we place some logic here instead of just returning tags. I would suggest a return type of List (for polymorphism) and then we can create a new empty list (ArrayList works) and add all tags to it; if tags is null, its returned an empty list. That would help avoid the NPE risk.

Reply:
Good point. 
Now it creates a copy of the list and returns it.